source/common/router/config_impl.h

```diff
@@ -1543,7 +1545,7 @@ class RouteMatcher {
 };
 
 /**
- * Shared part of the route configuration implementation.
+ * Shared part of the global route configuration implementation.
```

Comment:
All these changes in this file seems are unnecessary. (The typo fix could be kept.) To make PR clean could simplify the reviewing.